sdk/test-utils/perfstress/src/index.ts

```diff
@@ -6,4 +6,5 @@ export * from "./options";
 export * from "./policy";
 export * from "./parallel";
 export * from "./program";
+export * from "./testProxyHttpClient";
```

Comment:
If we add the `PerfStressTest.configureClient()` method we should be able to remove this.